e2e-pw/src/oss/poms/grid/index.ts

```diff
@@ -92,4 +97,14 @@ class GridAsserter {
       { timeout: 2000 }
     );
   }
+
+  async waitForGridToLoad() {
+    return this.gridPom.page.waitForSelector("[data-cy=looker]", {
+      timeout: 2000,
+    });
+  }
+
+  async delay(ms: number) {
+    new Promise((resolve) => setTimeout(resolve, ms));
```

Comment:
@sashankaryal @manivoxel51 I added this delay function, because I tried to wait for looker to load or search for certain attributes, but they don't really work. 
There are two specific cases that I needed this: 
1) wait for the js render update on canvas
2) wait for samples to reload (using result come back from aggregation)